glpi-project/glpi
# Fix inconsistency on date fields and add index on name fields

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

1. Add a checks on date fields consistency (`date_creation` and `date_mod` should not be used separately).
2. Fields that store a valuable date should be named `date`.
3. Add a check to ensure `name` fields are indexed (as this is the default field used for sorting lists).
